internal/endpoints/payloads_test.go

```diff
@@ -315,5 +342,53 @@ var _ = Describe("RequestIdPayloads", func() {
 				Expect(respData.Durations["puptoo:undefined"]).To(Equal("00:00:09.970000"))
 			})
 		})
+
+		reqIdPayloads = getFourReqIdPayloads(requestId, "1")
+		Context("Get to /payloads/{request_id} Verbosity 1", func() {
+			It("should pass the data forward", func() {
+				req, err := makeTestRequest(fmt.Sprintf("/api/v1/payloads/%s", requestId), query)
+				Expect(err).To(BeNil())
+
+				reqIdPayloadData = reqIdPayloads
+				handler.ServeHTTP(rr, req)
+				Expect(rr.Code).To(Equal(200))
+				Expect(rr.Body).ToNot(BeNil())
+
+				var respData structs.PayloadRetrievebyID
+
+				readBody, _ := ioutil.ReadAll(rr.Body)
+				json.Unmarshal(readBody, &respData)
+
+				Expect(respData.Data[0].Service).To(Equal(reqIdPayloads[0].Service))
+				Expect(respData.Data[0].InventoryID).To(Equal(reqIdPayloads[0].InventoryID))
+				Expect(respData.Data[0].CreatedAt).To(Equal(reqIdPayloads[0].CreatedAt))
```

Comment:
This is going to fail in github actions. Any time.Time values should have `.String()` on the end
example: `Expect(respData.Data[0].CreatedAt.String()).To(Equal(reqIdPayloads[0].CreatedAt.String())`